srcpkgs/jmol/template

```diff
@@ -0,0 +1,28 @@
+# Template for 'jmol'
+pkgname=jmol
+version=14.29.17
+revision=1
+noarch=yes
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="http://jmol.sourceforge.net/"
+depends="virtual?java-environment"
+license="LGPL-3"
+hostmakedepends="unzip"
+short_desc="An open-source Java/JavaScript-based molecule viewer"
+distfiles="${SOURCEFORGE_SITE}/projects/jmol/files/Jmol/Version%20${version%.*}/Jmol%20${version}/Jmol-${version}-binary.tar.gz http://jmol.sourceforge.net/images/Jmol_icon_128.png"
+skip_extraction="Jmol_icon_128.png"
+checksum="5347e0df0781620d8e1cb7b933cd76ca04d19aef57ceea1b153f2988e25eddb6
+ d84e5e4b5c9b440b3b90432db87e0d4bd5c8237d7d2891c174fac71f0a5a2127"
+
+do_install() {
+	mkdir -p "${DESTDIR}"/usr/{bin,share/${pkgname}}
```

Comment:
you can use `vmkdir`

Reply:
Hmm, I tried doing this and I got this error: https://gist.github.com/fusion809/1106863300662238550e233275eb57e3, from Travis. All I did was replace mkdir -p with vmkdir in the above line.

Reply:
Nvm, I figured out how to use it by looking at common/environment/setup/install.sh.